books/philosophy/bake

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '9.1.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :philosophy) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'references',
+      uuid_key: '.references',
+      section_selector: 'section.references',
+      include_intro_page: true
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'further-reading',
+      uuid_key: '.further-reading',
+      section_selector: 'section.further-reading',
+      include_intro_page: true
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+
+    chapter.search('.review-questions').exercises.each do |exercise|
+      BakeNumberedExercise.v1(exercise: exercise, number: exercise.count_in(:chapter))
+    end
+    MoveExercisesToEOC.v2(
+      chapter: chapter,
+      metadata_source: metadata,
+      klass: 'review-questions'
+    )
+    chapter.figures.each do |figure|
+      BakeFigure.v1(figure: figure,
+                    number: "#{chapter.count_in(:book)}.#{figure.count_in(:chapter)}")
+    end
+    chapter.tables('$:not(.unnumbered)').each do |table|
+      BakeNumberedTable.v1(table: table,
+                           number: "#{chapter.count_in(:book)}.#{table.count_in(:chapter)}")
+    end
+  end
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[read-philosopher
+                think-philosopher
+                write-philosopher
+                philo-connections
+                time-line]
+  )
+  book.notes.each do |note|
+    BakeNoteIFrames.v1(note: note)
+  end
```

Comment:
`BakeNoteIFrames` should already be called by `BakeAutotitledNotes` -- are `iframes` unbaked if this is removed?

Reply:
oop! didn't realize that

Reply:
oh it looks like they are unbaked when i remove `BakeNoteIFrames`
i'll look to see why

Reply:
excuse the small screen in the screenshot lol 
<img width="1151" alt="Screen Shot 2021-07-19 at 1 48 21 PM" src="https://user-images.githubusercontent.com/24909007/126225375-4e01ab53-4366-49de-810a-55cb938c3a9c.png">

i thought it was unbaked at first but it looks like there's an extra `<div class="os-has-iframe os-has-link" data-type="alternatives">` in the expected file on the left which was baked with `BakeNoteIFrames`  and `BakeAutotitledNotes` so I'm assuming thats why there is a double element

Reply:
I just need to update my spec now